src/sortStudents.ts

```diff
@@ -1,16 +1,63 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export enum SortOrder {
+  Ascending = 'asc',
+  Descending = 'desc',
+}
+
+function findAvgGrade({ grades }: Student): number {
+  return grades.reduce((total, grade) => total + grade, 0) / grades.length;
+}
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const sortedStudents = [...students];
+
+  sortedStudents.sort((studentA, studentB) => {
```

Comment:
```suggestion
  return [...students].sort((studentA, studentB) => {
```
Can be returned immediately.